packages/core/src/content/locales.ts

```diff
@@ -1,38 +1,47 @@
 /**
- * Represents the response structure for a query that retrieves a locale.
+ * Represents the locale entity.
+ */
+export type Locale = {
+  /** The unique identifier of the locale. */
+  id: string;
+  /** The label of the locale. */
+  label: string;
+};
+
+/**
+ * Represents a locale query response.
```

Comment:
Actually not exactly `locale query response`, since we have an interface below that represents `locale query response`
May be can be updated to describe like that?
```suggestion
 * A locale item included in a locale query response.
```